app/uk/gov/hmrc/serviceconfigs/connector/SlackNotificationsConnector.scala

```diff
@@ -99,6 +101,31 @@ final case class SlackNotificationRequest(
 object SlackNotificationRequest {
   implicit val writes: OWrites[SlackNotificationRequest] = Json.writes[SlackNotificationRequest]
 
+
+  def downstreamMarkedForDecommissioning(channelLookup: GithubTeam, eolRepository: String, eol: Instant, impactedRepositories: Seq[String]): SlackNotificationRequest = {
+    val utc = ZoneId.of("UTC")
+    val blocks = Seq(
+      Json.obj(
+        "type" -> JsString("section"),
+        "text" -> Json.obj(
+          "type" -> JsString("mrkdwn"),
+          "text" -> JsString(s"`$eolRepository` has been marked with an end of life date of `${eol}`." +
+            s"\n\nThe following services may have a dependency on this repository and may be impacted past this date: ${impactedRepositories.mkString("\n")}" +
+            s"\n\nIf this was a mistake, please contact #team-platops")
```

Comment:
Not sure we need this line - we should be confident in our notifications, and people will reach out to PlatOps if they're confused anyway